README.md

```diff
@@ -41,6 +41,12 @@ Code for specific tasks like
 
 etc
 
+In order to create your own algorithm to test:
+
+1. Create <your_algo>.py and put it in one of the specific task folders in perception/tasks.
+
+2. Create a class which extends the TaskPerceiver class. perception/tasks/TaskPerceiver.py includes a template with documentation for how to do this.
+
```

Comment:
3. Add to the ALGOS dict

Reply:
That step is specified in the running vis section below this one